buf/registry/module/v1/module_service.proto

```diff
@@ -115,10 +115,10 @@ message CreateModulesRequest {
       max_len: 100
     }];
     // The module's visibility.
-    ModuleVisibility visibility = 3 [
-      (buf.validate.field).required = true,
-      (buf.validate.field).enum.defined_only = true
-    ];
+    //
+    // If not specified, it uses the value from the instance or organization visibility control,
```

Comment:
Is this value (or the ability to set it) exposed via the api in any way? If not, should it be?

Reply:
Yes, this is the `CreateModulesRequest`, akin to the field in the `registryv1alpha1.RepositoryService.CreateRepositoryByFullNameRequest.Visibility` [here](https://github.com/bufbuild/buf/blob/b8451938bc0aede91789a773daafbdff1680f77c/proto/buf/alpha/registry/v1alpha1/repository.proto#L272).

They can also set this value both in the CLI using `--visibility` flag, or in the UI, which will both default to empty, so the default will be selected in the backend.

Reply:
Sorry I think I was unclear. By "this value" I meant the default from the instance or org level